src/edu/csus/ecs/pc2/core/imports/clics/CLICSLanguage.java

```diff
@@ -0,0 +1,45 @@
+// Copyright (C) 1989-2022 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.core.imports.clics;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import edu.csus.ecs.pc2.core.model.JSONObjectMapper;
+
+/**
+ * A JSON representation for the CLICS API languages json data.
+ * 
+ * @author Douglas A. Lane <pc2@ecs.csus.edu>
+ */
+public class CLICSLanguage {
+
+    // languages data = {name=C, id=c}
+
+    /**
+     * short name, ex python3
+     */
+    @JsonProperty
+    private String id;
+
+    /**
+     * Language title, ex Java or Python 3
+     */
+    @JsonProperty
+    private String name;
+
+    public String getId() {
```

Comment:
The above set of fields is missing a number of Language attributes defined in the CLICS 2202-07 spec, including `entry_point_required`, `entry_point_name`, `extensions`, `compiler`, and `runner`.    It's also missing definitions for _all _ of the fields defined in the `Command` object set (under Languages) in the spec.  Since_ we are trying to bring PC2 up to CLICS 2202-07 compliance, I think this class should include all the fields defined in that spec.

Note that some of these missing fields would provide support for a related issue: comparing the language arguments between a Shadow and a Primary CCS (e.g. the spec provides support for specifying the command-line arguments to be passed to a compiler).

Reply:
I agree that those fields should exist, but out_ of scope  for comparing the data from the event feed.
Copied into new issue #559

Reply:
I'm not sure why you think this is "out of scope for comparing the data from the event feed".   What happens if the event feed includes those fields defined in the spec?

Reply:
Then for this issue they are ignored.   Waiting for an agreement on the content/layout for the comparison,
after that the out of scope fields can be added.